server/imports/fixtures/products.js

```diff
@@ -212,10 +213,13 @@ export default function () {
     max: 12.99
   };
 
+  const productTitle = faker.commerce.productName();
+
   const product = {
-    title: faker.commerce.productName(),
+    title: productTitle,
```

Comment:
All the other fields are being populated on the fly here, why move the productTitle into it's own `const`?

Reply:
Nevermind... didn't see it was being used in the `handle`